### PR TITLE
DOC-1044: RC - Add shard descriptions to Create Flexible topic

### PR DESCRIPTION
**Pull request by lanceleonard**
- Adding shard descriptions & other minor edits

### CODE REVIEW

**Comment by lanceleonard** [comment]
A [staged preview](https://docs.redis.com/staging/jira-doc-1044/rc/subscriptions/create-flexible-subscription/#shard-types) is available.

File: content/rc/subscriptions/create-flexible-subscription.md [hunk 1]
```diff
@@ -142,8 +140,22 @@ The **Review & Create** tab provides a cost estimate for your Flexible plan:
...
-{{<image filename="images/rc/button-subscription-create.png" alt="Select Create subscription to create your new subscription." >}}{{< /image >}}
+{{<image filename="images/rc/button-subscription-create.png" width="140px" alt="Select Create subscription to create your new subscription." >}}{{< /image >}}
 
 Note that subscriptions are created in the background.  While they are provisioning, you aren't allowed to make changes.  (The process generally takes 10-15 minutes.)
 
 Use the **Subscriptions list** to check the status of your subscription.  You will also receive an email when your subscription is ready to use.
+
+### Shard types
+
+The shard types associated with your subscription depend on your database memory size and throughout requirements.
```
Comment (rrelledge) [inline content/rc/subscriptions/create-flexible-subscription.md:151]:
```suggestion
The shard types associated with your subscription depend on your database memory size and throughput requirements.  
```

**Review by rrelledge — APPROVED** [review]
Looks good to me.

**Review by kaitlynmichael — APPROVED** [review]
LGTM

File: content/rc/subscriptions/create-flexible-subscription.md (hunk 1 above)
Comment (lanceleonard) [reply content/rc/subscriptions/create-flexible-subscription.md]: Fixed.

**Pull request merged**